libs/src/api/serviceProvider.js

```diff
@@ -47,7 +47,7 @@ class ServiceProvider extends Base {
     const timings = await timeRequestsAndSortByVersion(
       creatorNodes.map(node => ({
         id: node.endpoint,
-        url: `${node.endpoint}/health_check/verbose`
+        url: `${node.endpoint}/health_check`
```

Comment:
i think we should make this libs change in the next phase once we've validated that the health check is now the same as verbose

Reply:
This PR was meant to be for the libs phase and the first phase is https://github.com/AudiusProject/audius-protocol/pull/1868.

Reply:
ohhh sorry i got confused between the two 🤦

Reply:
for phase 2 this looks good